src/main/webapp/scripts/view.js

```diff
@@ -78,7 +81,7 @@ function successfulApiCallback(data) {
  */
 function failedApiCallback(err) {
     console.error('execution error ', err);
-    updateDom(err, postResultContainer);
+    updateDom('An error occurred for posting this comment.', postResultContainer);
```

Comment:
changed this because it was giving the user long and confusing errors meant for the console when they did things like not having a video to comment on

Reply:
Actually the error gives some useful information like comments can't be empty, or the video isn't found etc. I think it is informative to show the user that piece?

Reply:
This is the one that bothered me, happens when a user didnt input a video. the first sentence is super helpful but the rest is ??? not even sure why there's all those tags, maybe something broke with the API because that's what console logs too. Maybe in the future it won't be a giant code block or we can write a function that converts errors like these into nice text that a user can read.
![Screenshot 2020-07-29 at 3 09 09 PM](https://user-images.githubusercontent.com/15972777/88858840-7cb4ea00-d1ad-11ea-8017-88b301cdd82b.png)